api/src/test/java/org/openmrs/api/AdministrationServiceTest.java

```diff
@@ -594,6 +618,19 @@ public void getGlobalPropertyValue_shouldReturnValueInTheSpecifiedDoubleType() {
 		Assert.assertEquals(1234.54, retValue);
 	}
 	
+	/**
+	 * @see AdministrationService#getGlobalPropertyValue()
+	 */
+	@Test
+	public void getGlobalPropertyValue_shouldFailIfDefaultValueIsNull() {
```

Comment:
this test is also a great candidate for being written in the new `AdministrationServiceUnitTest` since no DB is needed

Reply:
@teleivo ok I will add it to AdministrationServiceUnitTest.